INSTALL.md

```diff
@@ -320,6 +320,10 @@ with `vcpkg install opentelemetry-cpp` and follow the then displayed
 descriptions. Please see the vcpkg project for any issues regarding the
 packaging.
 
+If you are using [alpine linx](https://www.alpinelinux.org/) you can install 
```

Comment:
There is a trailing space, making the format CI fail.

Also, `alpine linux` for the link name ?